### ISSUE TITLE
[Suggestion] support `Rotate tile` and supply Plugin/extension  System

### ISSUE THREAD

**Issue by finscn** [deepnight/ldtk#616]
I found there is mirror function of tile , but no  Rotate By 90deg.
I hope could support this feature.

And I'm a TiledMap user.  It supplies a extension  System by javascript 
I hope LDTK could do .

Thanks

**Comment by Cammin** [comment]
Looks like it's on the roadmap for 1.0 at #207

**Comment by finscn** [comment]
> Looks like it's on the milestone for 1.0 at #207

Yes , the first one is in the roadmap, but I've not found  the 2nd & 3rd .

**Comment by deepnight** [comment]
Hi, thanks for your suggestions :)

2nd is quite close from the #68 , even though I'm not sure yet to support properties per sprite object, as it would be highly redundant with Entities, especially since they fully support Tiles in 1.0 .

3rd is a cool idea, but requires lots of design thinking to determine the needs and limitations.

**Comment by finscn** [comment]
@deepnight   ,  I create some  Extensions of TiledMap  , example : 

*  topological sorting plugin :  I need to create an isometric map , the sorting is a challenge. TiledMap only supports `sort by Y, sort by custom` . So  , I write a plugin to sort objects in the object-tree by  topological sorting algorithm. 
* auto-name image-objects :  I don't want to named every object's name manually .  I write a plugin to auto-name the image-object with image file name.
* hide/display/remove objects by any custom property
* custom export file format

The above is just example,  maybe  it's not  suitable to LDtk .

**Comment by unitycoder** [comment]
(was googling about plugin/extension systems.. this was only hit)

I would use it for adding generator plugins.
Currently i'm trying to do it by outputting / modifying .ldtk file.. (and it kind of works)

but would be better, if could create your own plugins,
and then inside LDTK could press custom "generate" button (to for example: generate tiles inside selection or current level etc).

looks like this is slightly related #423
Having intgrid import method would help,
but nice if can also import intGridValues (i'm currently generating them externally).
And nicer, if can refresh that intgrid (or autorefresh, if the csv file is modified externally).
